environments/ithc/ithc.tfvars

```diff
@@ -1154,5 +1154,20 @@ frontends = [
     custom_domain    = "et-sya.ithc.platform.hmcts.net"
     backend_domain   = ["firewall-nonprodi-palo-cftithc.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-ithc-platform-hmcts-net"
+  },
+  {
+    product          = "dss-update-case"
+    name             = "dss-update-case"
+    custom_domain    = "dss-update-case.ithc.platform.hmcts.net"
+    backend_domain   = ["firewall-nonprodi-palo-cftithcappgateway.uksouth.cloudapp.azure.com"]
```

Comment:
I think this should be `"firewall-nonprodi-palo-cftithc.uksouth.cloudapp.azure.com"`

Reply:
thanks @reespozzi , I have done the changes